test/server/cards/15.4-TToTS/MeticulousScout.spec.js

```diff
@@ -0,0 +1,264 @@
+            });
+
+            this.player2.pass();
+            this.player1.pass();
+            this.player1.clickPrompt('Take 1 Honor from opponent');
+
+            expect(this.player1).toHavePrompt('Action Window');
+            this.player1.clickCard(this.scout);
+            expect(this.player1).toHavePrompt('Action Window');
+        });
+
+        it('should trigger if you have gained 2 honor this phase', function() {
+            this.noMoreActions();
+            this.initiateConflict({
+                attackers: [this.scout],
+                defenders: [],
+                type: 'military'
+            });
+
+            this.player2.pass();
+            this.player1.pass();
+            this.player1.clickPrompt('Gain 2 Honor');
+
+            expect(this.player1).toHavePrompt('Action Window');
+            this.player1.clickCard(this.scout);
+            expect(this.player1).toHavePrompt('Choose a province');
+
+        });
+
+        it('should let you choose a province controlled by the conflict loser regardless of revealed or not', function () {
+            this.noMoreActions();
+            this.initiateConflict({
+                type: 'military',
+                attackers: [this.scout],
+                ring:'air',
+                defenders: [],
+                province: this.manicured
+            });
+            this.noMoreActions();
+            this.player1.clickPrompt('Gain 2 Honor');
+
+            this.player1.clickCard(this.scout);
+            expect(this.player1).toBeAbleToSelect(this.nightRaid);
+            expect(this.player1).toBeAbleToSelect(this.manicured);
+            expect(this.player1).toBeAbleToSelect(this.shameful);
+            expect(this.player1).toBeAbleToSelect(this.pilgrimage);
+            expect(this.player1).toBeAbleToSelect(this.p2_Stronghold);
+
+            expect(this.player1).not.toBeAbleToSelect(this.p1_1);
+            expect(this.player1).not.toBeAbleToSelect(this.p1_2);
+            expect(this.player1).not.toBeAbleToSelect(this.p1_3);
+            expect(this.player1).not.toBeAbleToSelect(this.p1_4);
+            expect(this.player1).not.toBeAbleToSelect(this.p1_Stronghold);
+        });
+
+        it('should dishonor (blanking) and reveal the province - should not trigger reveal abilities', function () {
+            this.noMoreActions();
+            this.initiateConflict({
+                type: 'military',
+                ring:'air',
+                attackers: [this.scout],
+                defenders: [],
+                province: this.manicured
+            });
+            this.noMoreActions();
+            this.player1.clickPrompt('Gain 2 Honor');
+
+            this.player1.clickCard(this.scout);
+            this.player1.clickCard(this.nightRaid);
+            expect(this.nightRaid.isDishonored).toBe(true);
+            expect(this.nightRaid.facedown).toBe(false);
+            expect(this.player2).not.toHavePrompt('Triggered Abilities');
```

Comment:
You'll probably want to use Khan's Ordu instead of Night Raid based on the tests you're running (Night Raid can't trigger outside of a conflict because there are no attacking characters)